src/domains/dashboard/components/node-view/node-view.tsx

```diff
@@ -69,7 +71,13 @@ const SubSection = memo(({
             .map(parseChartString)
             .map((attributes: Attributes | null) => attributes && (
               <ChartWrapper
-                attributes={{ ...attributes, host }}
+                attributes={
+                  {
+                    ...attributes,
+                    host,
+                    ...(attributesOverrides ? attributesOverrides[attributes.id] : {}),
```

Comment:
could it be placed in function?

Reply:
Could be, I just followed @jacekkolasa 's approach and decided to stay consistent:
```
 // add commonMin/commonMax attributes only if they are set
                ...(commonMin ? { commonMin } : {}),
                ...(commonMax ? { commonMax } : {}),
                ...(attributesOverrides ? attributesOverrides[chart.id] : {}),
```